install_config/configuring_kuryrsdn.adoc

```diff
@@ -109,6 +108,9 @@ Kuryr-Kubernetes as the network plug-in:
  kuryr_openstack_service_subnet_id=3b31a106-4084-4db9-bc0c-00b97afe186e
 ----
 
+You'll also need to specify an OpenStack cloud provider as described in the
+xref:configuring_openstack.adoc#openstack-configuring-masters-ansible[OpenStack configuration documentation].
```

Comment:
I think that the link needs to be formatted as xref:../

Please check that the link works in a preview build.

Reply:
Hi! I *think* this works; it worked for me in a local checkout.

Reply:
@tzumainn Okay, thanks! Also, we'll need to squash the 2 commits into 1 commit per our publishing process. Let me know if you need help with that.

Reply:
Ah, wasn't sure what the standard was. I've squashed the commits, thanks!